src/main/java/com/iopipe/plugin/profiler/ProfilerExecution.java

```diff
@@ -209,6 +243,21 @@ final void __post()
 				Base64.getMimeEncoder().encodeToString(fexported) +
 				"\n====\n");
 			
+			// Dump a local copy? This is used for debugging snapshots
```

Comment:
Why is this a question here?

Reply:
This is optional and only dumps a file if the given environment variable is set, so it is never always a true case (and likely for the most case will always be a false case).

Reply:
Ah, that's useful. Update the comment to be a bit clearer? (what you just said made into something comment-y?). "If this enviroment variable is set, will dump a file, useful for debugging" perhaps?

Reply:
Oh I like that, along with the `PROFILER.md` documentation.